app/models/report_a_problem_ticket.rb

```diff
@@ -34,7 +34,7 @@ def spam?
 private
 
   def ticket_details
-    %i[what_wrong what_doing path user_agent javascript_enabled referrer source page_owner].each_with_object({}) do |field, details|
+    %i[what_wrong what_doing path user_agent javascript_enabled referrer source page_owner].index_with({}) do |field, details|
```

Comment:
Do you need the ({}) after `index_with` ?
https://apidock.com/rails/v6.0.0/Enumerable/index_with

Reply:
Hm i think it depends what you're iterating over. Seems like yes in your case. But I've had to drop that from updating a similar thing

Reply:
For reference: https://github.com/alphagov/release/pull/667/commits/1f71ec9a9af59d674e1f6324c13900c2cc64c305